bot/exts/fun/tic_tac_toe.py

```diff
@@ -77,50 +91,58 @@ def __init__(self, bot_user: discord.Member, symbol: str):
         self.symbol = symbol
 
     @staticmethod
-    async def get_move(board: dict[int, str], _: discord.Message) -> tuple[bool, int]:
+    async def get_move(board: dict[int, str], msg: discord.Message) -> tuple[bool, int, None]:
```

Comment:
Does the interface really has to be the same?
You're already checking if the player is an `AI` or a `Player`. So maybe just do something like:
```py
if isinstance(self.current, AI):
    pos = await self.current.get_move(self.board, board)
    interaction = None
else:
    move = await self.current.get_mode(self.board, board)
    if move is None:
        # handle timeout
    pos, inter = move
```